js/ui/overlay/ui.overlay.js

```diff
@@ -720,6 +720,21 @@ const Overlay = Widget.inherit({
         }
     },
 
+    _toggleFocusClass(isFocused, $element) {
+        this.callBase(isFocused, $element);
```

Comment:
let's get rid of the second parameter in this function and just pass `arguments` to the callbase. it's better for extensibility